optimum/commands/export/openvino.py

```diff
@@ -375,16 +359,22 @@ def run(self):
                     "Dataset is required for full quantization. Please provide it with --dataset argument."
                 )
 
-            quantization_config = {
-                "weight_format": self.args.quant_mode,
-                "activation_format": self.args.quant_mode,
-                "bits": 8,
-                "sym": self.args.sym or False,
-                "dataset": self.args.dataset,
-                "num_samples": self.args.num_samples,
-                "smooth_quant_alpha": self.args.smooth_quant_alpha,
-                "trust_remote_code": self.args.trust_remote_code,
-            }
+            if self.args.quant_mode == "nf4_f8e4m3":
+                wc_config = prepare_wc_config(self.args, _DEFAULT_4BIT_CONFIG)
+                wc_config["weight_format"] = "nf4"
```

Comment:
We need to set FP8 as a backup precision for the layers that are not in `nf4` by default.

Reply:
I will need to prepare this feature in NNCF first

Reply:
Implemented a workaround by providing a "none" backup precision. This keep sensitive weights in original precision after weight compression step. They will later be quantized to FP8 by quantization step.